lua/lsp/init.lua

```diff
@@ -153,4 +143,29 @@ function M.setup(lang)
   end
 end
 
+function M.global_setup()
```

Comment:
I like this, to make naming more consistent with builtins and my changes in 1406, are you ok renaming this to `setup` ?

Reply:
yeah I just wanted to avoid confusion about what this is actually doing, because it's just setting `nlsp`, `null-ls` and some configurations. There's still the actual function `setup()` that's language specific and needs to be explicitly called.

so what do you think then?